themes/default/content/resources/infrastructure-as-code-google-cloud/index.md

```diff
@@ -0,0 +1,53 @@
+---
+preview_image: ""
+hero:
+  image: /icons/containers.svg
+  title: Getting Started with Infrastructure as Code on Google Cloud
+main:
+  duration: 60 minutes
```

Comment:
how long is this workshop supposed to be. I didnt see it in the issue, so I put this for 60 mins. Is that correct?

Reply:
Yep! That's correct